molo/core/wagtail_hooks.py

```diff
@@ -65,3 +65,23 @@ def register_import_menu_item():
         urlresolvers.reverse('import-from-git'),
         classnames='icon icon-download',
     )
+
+
+class LanguageSummaryItem(SummaryItem):
+    order = 500
+    template = 'admin/site_languages_summary.html'
+
+    def get_context(self):
+        languages = SiteLanguage.objects.all()
+        return {
+            'summaries': [{
+                'language': l.get_locale_display(),
+                'total': Page.objects.filter(
+                    languages__language__id=l.id, live=True).count()
```

Comment:
I don't think we need `live=True` here. I think we should count all pages, including drafts